src/main/kotlin/com/gamingbacklog/api/gamingbacklogapi/controllers/UserController.kt

```diff
@@ -0,0 +1,57 @@
+package com.gamingbacklog.api.gamingbacklogapi.controllers
+
+import com.gamingbacklog.api.gamingbacklogapi.models.User
+import com.gamingbacklog.api.gamingbacklogapi.models.requests.UserRequest
+import com.gamingbacklog.api.gamingbacklogapi.services.UserService
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+
+@RestController
+@RequestMapping("/users")
+class UserController(private val userService: UserService) {
+
+  @CrossOrigin(origins = ["http://localhost:3000"])
+  @GetMapping("/")
+  fun getAllUsers(): ResponseEntity<List<User>> {
```

Comment:
We should probably have a `UserResponse` model as well (I don't think we want to be returning plain text passwords from the backend :') )